### PR TITLE
feat: added favicon

### PR DESCRIPTION
**Pull request by TJSteel**

### CODE REVIEW

File: logorama/_includes/head.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
   <meta property="og:image" content="https://logorama.github.io/assets/images/logos/logo-36.png">
   <meta property="og:image" content="https://logorama.github.io/assets/images/logos/logo-52.png">
   <meta property="og:image" content="https://logorama.github.io/assets/images/logos/logo-53.png">
+  <link rel="icon" type="image/gif" href="/assets/images/LogoRama_favicon.ico">
```
Comment (jbampton) [inline logorama/_includes/head.html:11]: Can we just call it -> favicon.ico -> all lowercase ?

**Review by jbampton — CHANGES_REQUESTED** [review]

**Comment by jbampton** [comment]
Needs a rebase too

**Comment by jbampton** [comment]
Hey @TJSteel can you rebase again ?

I want to merge this PR next.

**Comment by TJSteel** [comment]
> Hey @TJSteel can you rebase again ?
> 
> I want to merge this PR next.

just done this now, would a rebase and merge not have done the same thing though?

**Pull request merged**